src/main/kotlin/lotto/model/game/Lottos.kt

```diff
@@ -0,0 +1,56 @@
+package lotto.model.game
+
+import lotto.model.result.Coincidence
+import java.math.BigDecimal
+import java.math.RoundingMode
+
+class Lottos(val lottos: List<Lotto>) {
+    constructor(count: Int) : this(createLottos(count))
+
+    fun getEarningRate(winningLotto: Lotto): BigDecimal {
+        val totalPrizeMoney = getTotalPrizeMoney(winningLotto)
+        val budgetMoney = (lottos.size * COST_PER_ONE_LOTTO).toBigDecimal()
+
+        return totalPrizeMoney.divide(budgetMoney, TWO_DECIMAL_PLACES, RoundingMode.FLOOR)
+    }
+
+    fun getCoincidenceCount(coincidence: Coincidence, winningLotto: WinningLotto): Int {
+        return lottos.count { coincidence == it.getResult(winningLotto) }
+    }
+
+    fun add(other: Lottos): Lottos {
+        val newLottos = lottos.toMutableList()
+        other.lottos.forEach { newLottos.add(it) }
+
+        return Lottos(newLottos)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return Lottos(lottos + other.lottos)
```